src/content/en/tools/chrome-devtools/javascript/overview.md

```diff
@@ -0,0 +1,140 @@
+project_path: /web/_project.yaml
+book_path: /web/tools/_book.yaml
+description: An overview of debugging JavaScript using Chrome DevTools.
+
+{# wf_updated_on: 2016-11-04 #}
+{# wf_published_on: 2016-11-04 #}
+
+# Overview of Debugging JavaScript in Chrome DevTools {: .page-title }
+
+{% include "web/_shared/contributors/kaycebasques.html" %}
+
+This page provides a conceptual overview of what it means to debug JavaScript
```

Comment:
'what it means to debug' I'm worried this phrase might be confusing to non-native readers.